src/hooks/useUniswapPools.js

```diff
@@ -146,23 +147,23 @@ async function getBulkPairData(pairList, ethPrice, ethPriceOneMonthAgo) {
           let oneDayHistory = oneDayData?.[pair.id];
           if (!oneDayHistory) {
             const newData = await uniswapClient.query({
-              fetchPolicy: 'network-only',
+              fetchPolicy: 'no-cache',
               query: UNISWAP_PAIR_DATA_QUERY(pair.id, b1),
             });
             oneDayHistory = newData.data.pairs[0];
```

Comment:
there was one time when I saw: `error in getBulkPairData [TypeError: undefined is not an object (evaluating 'newData.data.pairs')]` 
haven't been able to reproduce, but we should add some optional chaining